build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -392,9 +392,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="2048,512,1024,1024,1024,1024,1024,"
-CONFIG_FLASH_PART_TYPE="kernel,none,bin,bin,bin,bin,smartfs,"
-CONFIG_FLASH_PART_NAME="kernel,app,micom,micom,wifi,wifi,userfs,"
+CONFIG_FLASH_PART_SIZE="2048,512,1024,1024,1024,1024,1024,256,256,"
```

Comment:
for ramdump, 256KB is definitely not enough. Is it possible to check that functionality? If it isn't, we had better remove this partition.

Reply:
I have added the condition to check dump file size with partition size, so it will taken care at crashdump.c